Gom4ziz/Source/Presenter/Main/ReviewedArtworkList/ReviewedArtworkListViewController.swift

```diff
@@ -0,0 +1,110 @@
+//
+//  ReviewedArtworkListViewController.swift
+//  Gom4ziz
+//
+//  Created by 이가은 on 2022/12/01.
+//
+
+import UIKit
+
+import RxCocoa
+import RxDataSources
+import RxSwift
+
+final class ReviewedArtworkListViewController: UIViewController {
+    
+    private var tableView = UITableView()
+    private let viewModel: ReviewedArtworkListViewModel
+    private let userId: String
+    private let artworkCount: Int
```

Comment:
[수정 고려]
userId와 artworkCount는 어떤 유저의 작품들을 가져올지 결정하기 위해서 필요한 정보들이죠? 이런 정보들은 ReviewedArtworkListViewModel 에서 그냥 User 타입으로 가지고 있는게 좋을 것 같아요!

```suggestion
ReviewedArtworkListViewModel {
  private let user: User
  
  init(user: User) {
    self.user = user
  }
}
```

Reply:
그러면 ViewController는 ViewModel만 받아도 모든 로직을 실행할 수 있겠죠.

Reply:
[a301c05](https://github.com/DeveloperAcademy-POSTECH/MacC_Team_Beartear/pull/171/commits/a301c05096b7ba81d49bacef30b8fb150dbddd5d#:~:text=private%20let%20viewModel,%7D)에서 수정했습니다 감사합니다 !